src/huggingface_hub/file_download.py

```diff
@@ -48,6 +48,34 @@
 except importlib_metadata.PackageNotFoundError:
     pass
 
+_pydot_version = "N/A"
```

Comment:
@LysandreJik It feels a bit weird to have these checks here. At the same time, having them here is consistent. Should we have them here or in the keras code?